### PR TITLE
fixing infinite loop that happens while changing assets

### PR DESCRIPTION
**Pull request by maciejt10c**
Task: [ACD-647](https://10clouds.atlassian.net/browse/ACD-647)

[ACD-647]: https://10clouds.atlassian.net/browse/ACD-647?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: frontend/src/components/editables/assets/AssetEditor.tsx [hunk 1]
```diff
@@ -161,20 +162,22 @@ export function AssetEditor({ assetType }: { assetType: AssetType }) {
       });
     } else {
       //For id === 'new' This will get a default new asset
+      console.log('USE EFFECT 1');
```
Comment (aleksanderfret) [inline frontend/src/components/editables/assets/AssetEditor.tsx:165]: leftover?

File: frontend/src/components/editables/assets/AssetEditor.tsx [hunk 2]
```diff
@@ -161,20 +162,22 @@ export function AssetEditor({ assetType }: { assetType: AssetType }) {
...
     }
-  }, [copyId, editableObjectType, id, setLastSavedSelectedAsset, setSelectedAsset, searchParams]);
+  }, [copyId, editableObjectType, id, searchParams, setLastSavedSelectedAsset, setSelectedAsset]);
 
   // Acquire the initial object
   useEffect(() => {
     getInitialAsset();
-
     return () => {
+      console.log('test');
```
Comment (aleksanderfret) [inline frontend/src/components/editables/assets/AssetEditor.tsx:180]: leftover?

File: frontend/src/components/editables/assets/AssetEditor.tsx [hunk 3]
```diff
@@ -148,6 +148,7 @@ export function AssetEditor({ assetType }: { assetType: AssetType }) {
       handleDeleteWithInteraction(id);
     }
   }
+  const [hasCore, setHasCore] = useState(false);
```
Comment (aleksanderfret) [inline frontend/src/components/editables/assets/AssetEditor.tsx:151]: do we need to change the place when it is defined. As it is exactly the same and only causes additional divs

**Review by aleksanderfret — APPROVED** [review]

File: frontend/src/components/editables/assets/AssetEditor.tsx (hunk 3 above)
Comment (maciejt10c) [reply frontend/src/components/editables/assets/AssetEditor.tsx:151]: good point, will revert it

**Pull request merged**